spec/models/video_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+
+describe Video do
+  it { should belong_to(:category)}
+  it { should validate_presence_of(:title)}
+  it { should validate_presence_of(:description)}
+
+  describe "search_by_title" do
```

Comment:
The issue is that I don't know what method is that - class or instance? `describe "search_by_title"` Its easier to annotate your methods and driving RSpecs style on describing. Prepending `.` would be a class method and `#` would be an instance method. Read in the link I mentioned.